src/Streams.sol

```diff
@@ -683,29 +736,29 @@ abstract contract Streams {
     /// @param newReceivers The list of the streams receivers of the account to be set.
     /// Must be sorted by the account IDs and then by the stream configurations,
     /// without identical elements and without 0 amtPerSecs.
-    /// @param maxEndHint1 An optional parameter allowing gas optimization, pass `0` to ignore it.
-    /// The first hint for finding the maximum end time when all streams stop due to funds
+    /// @param maxEndHints An optional parameter allowing gas optimization.
+    /// Pass a list of 8 zero value hints to ignore it, it's represented as an integer `0`.
```

Comment:
```suggestion
    /// Pass `0` to ignore it. 
```
That's too complicated in my opinion `list of 8 zero value hints` for the reader who only wants to skip it.

Reply:
Clarified.